tests/test_notifications.py

```diff
@@ -1234,7 +1230,9 @@ def test_separate_users(self):
         assert_equal([self.user_4._id], removed)
 
     def test_event_subs_same(self):
-        self.file_sub.email_transactional.extend([self.user_2, self.user_3, self.user_4])
+        self.file_sub.email_transactional.add(self.user_2)
```

Comment:
[self.file_sub.email_transactional.add(self.user_2, self.user_3, self.user4)](http://stackoverflow.com/questions/4959499/how-to-add-multiple-objects-to-manytomany-relationship-at-once-in-django?answertab=votes#tab-top)